KubeArmor/core/kubeUpdate.go

```diff
@@ -423,6 +428,7 @@ func (dm *KubeArmorDaemon) WatchK8sPods() {
 						}
 					}
 
+					pod.AlreadyDeployed = !updateAppArmor
```

Comment:
its coverage is too huge. KubeArmor runs on the environments with no LSM or some other LSMs except for AppArmor, which means that we will always see pod.AlreadyDeployed=true.

Reply:
When I tested the code, I think we don't need the AlreadyDeployed flag.
Without the flag, it works fine in the following environments.
- ubuntu 20.04 with docker, ubuntu 20.04 with containerd, ubuntu 18.04 with microk8s
- minikube with 4.19 kernel, ubuntu 18.04 (local), ubuntu 20.04 (local)

Reply:
I think we don't need this

Reply:
I will remove this but then we need to call LogFeeder.UpdateSecurity irrespective of this flag. I added this to make sure that MatchPolicy struct was added for already deployed containers as well. Because for already deployed pods we don't get their modified event because we never patch them.

Reply:
It seems that it works without policy updates.
I ran like this.
start KubeArmor -> deploy multiubuntu -> stop KubeArmor -> start KubeArmor -> ./test-scenarios-in-runtime.sh

Reply:
It does work but not always. At times it won't that happens when some pod added event is detected later than the added security policy event. In that case the containergroup for that pod is empty when the policy arrives so entry for them are not added but when the pod added event arrives it does not call the updatesecurity policy again so the logfeeder remains unaware of those containergroup matchpolicy structs. You can reproduce this behavior by repeating the above sequence. I added this line to cover that corner case.